lib/adapter.js

```diff
@@ -374,5 +392,56 @@ module.exports = (function () {
     return goDeep(original, mapperFn)
   }
 
+    /**
```

Comment:
Can this change be merged into upgrade/sailsjs branch? (Are your changes, a breaking change?)

Reply:
No, these are breaking changes as it involves `jsforce` upgrade. We can push to a new branch in either https://github.com/nutanix/sails-forcedotcom or https://github.com/shashankmeyar/sails-postgresql